test/jest-helpers.js

```
@@ -0,0 +1,183 @@
+const fs = require('fs');
+const path = require('path');
+
+// Custom matcher to verify that an array contains a value matching a regex or string
+expect.extend({
+  toContainMatch(received, expected) {
+    if (!Array.isArray(received)) {
+      throw new Error('toContainMatch can only be used on arrays');
+    }
+    const regex = typeof expected === 'string' ? new RegExp(expected.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')) : expected;
+    const stripAnsi = str => str.replace(/\u001b\[[0-9;]*m/g, '');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)